Configuration/Settings.yaml

```diff
@@ -1,5 +1,6 @@
 Flowpack:
   ElasticSearch:
+    version: 2.x
```

Comment:
I think it would be better to set the default value to 1.x. So the package behaviour won't change out of the box and you have to intentionally change the setting to get a different result.